src/cascadia/TerminalApp/PowershellCoreProfileGenerator.cpp

```diff
@@ -322,8 +323,8 @@ std::vector<TerminalApp::Profile> PowershellCoreProfileGenerator::GenerateProfil
         // This will turn the anchored default profile into "PowerShell Core Latest for Native Architecture through Store"
         // (or the closest approximation thereof). It may choose a preview instance as the "best" if it is a higher version.
         auto firstProfile = profiles.begin();
-        firstProfile->SetGuid(PowershellCoreGuid);
-        firstProfile->SetName(POWERSHELL_PREFERRED_PROFILE_NAME);
+        firstProfile->Guid(winrt::guid{ PowershellCoreGuid });

```

Comment:
This _shouldn't_ need the manual construction of a `winrt::guid`

Reply:
just change the type of powershellCoreGuid _to a `winrt::guid` already_